src/test/java/com/google/api/codegen/testdata/python_doc_message_library.baseline

```diff
@@ -1080,7 +1080,7 @@ class StreamShelvesResponse(object):
     Response message for LibraryService.StreamShelves.
 
     Attributes:
-      shelves (list[Union[dict|:class:`google.cloud.proto.example.library.v1.library_pb2.Shelf`]]): The list of shelves.
+      shelves (list[Union[dict|~.library_pb2.Shelf]]): The list of shelves.
```

Comment:
I think this is on me for giving you the wrong thing, but the format is `Union[dict, ~.library_pb2.shelf]` -- comma, not pipe. (This applies everywhere.)